Applications/FileManager/FilePathList.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <AK/FileSystemPath.h>
+#include <AK/String.h>
+#include <AK/Vector.h>
+
+/**
+ * A list of files and folder paths that might be copied and moved.
+ */
+class FilePathList {
+public:
+    /* Create a new FileSystemPath from a list of paths */
+    FilePathList(Vector<FileSystemPath> paths)
```

Comment:
Please pass heavy types by const reference to reduce copying:
```suggestion
    FilePathList(const Vector<FileSystemPath>& paths)
```